src/three-components/ModelUtils.ts

```diff
@@ -80,14 +90,30 @@ export const cloneGltf = (gltf: Gltf): Gltf => {
   const cloneBones: BoneMap = {};
   const cloneSkinnedMeshes: SkinnedMeshMap = {};
 
-  if (hasScene && hasSkinnedMeshes) {
+  if (hasScene) {
```

Comment:
Can we combine this with the same if statement above?

Reply:
Yeah, good call, I think we can merge the conditions.